include/micm/util/cuda_dense_matrix.hpp

```diff
@@ -50,7 +55,7 @@ namespace micm
     CudaDenseMatrix(std::size_t x_dim, std::size_t y_dim) requires(std::is_same_v<T, double>)
         : VectorMatrix<T, L>(x_dim, y_dim)
     {
-      micm::cuda::MallocVector(param_, this->data_.size());
+      micm::cuda::CHECK_CUDA_ERROR(micm::cuda::MallocVector(this->param_, this->data_.size()), __FILE__, __LINE__, "cudaMalloc");
       this->param_.number_of_grid_cells_ = x_dim;
     }
```

Comment:
Should this also include `micm::cuda::CHECK_CUBLAS_ERROR(cublasCreate(&(this->handle_)), __FILE__, __LINE__, "CUBLAS initialization failed...");`?

Reply:
Thanks for catching it. Eventually we may not want to create the cublas handle in this class but we can add it for consistency for now.